ODP-Runtime.md

```diff
@@ -124,61 +121,54 @@ Compliance
 
 In order to locate common resources, some commonly set configuration details are necessary. 
 
--   All environment variables discussed in this section MUST be set on all nodes.
-> Comment (SCG): We need to discuss the scope of these variables. Are they automatically available to every user that logs into the machine, or is there a standard location in which an "-env.sh" script can be found to set the environment variables?
-
--   An ODP Platform MUST set the JAVA\_HOME.
-> Comment (SCG): We had discussed this being available via the "hadoop" script, is this requiring it to be set globally? 
+-   An ODP Platform MUST either explicitly set `JAVA_HOME` or configure it in `hadoop-env.sh` and `yarn-env.sh`. In a future specification, `yarn-env.sh` will be removed.
 
 -   ODP Platforms MUST have all of the base Apache Hadoop components installed.
 
--   ODP Platforms MUST pass the Big Top smoke tests.
-> Comment (AFG) I assume this means the ODP specific Big Top tests rather than generic Apache Big Top ones.  This should be made explicit.
+-   ODP Platforms MUST pass the Apache Big Top 1.0.0 Hadoop smoke tests.
 
 -   ODP Platforms MUST NOT change public APIs, where an API is defined as either a Java API (aka "Apache Hadoop ABI") or a REST API. See the [Apache Hadoop Compatibility guidelines](http://hadoop.apache.org/docs/r2.7.1/hadoop-project-dist/hadoop-common/Compatibility.html#Java_Binary_compatibility_for_end-user_applications_i.e._Apache_Hadoop_ABI) for more information.
 
--   ODP Platforms MUST NOT modify version strings output by Hadoop components, such as those displayed in log files, or returnd via public API's.
+-   ODP Platforms MUST modify the version string output by Hadoop components, such as those displayed in log files, or returned via public API's such that they contain `-(vendor string)` where `(vendor string)` is an appropriate entry for the ODP Platform vendor in the output.
```

Comment:
I think this needs to be a bit more prescriptive. I added this because we ran into a problem with a partner that declared it a "bug" that our distribution presented the Ambari version string via their REST API as "1.7-IBM".   My main concern is that if we say MUST, then we should do so in a way that can prescribe to ISV's how to separate the version number from the identifying vendor information.   Such as -[A-Za-z_0-9] that matches a string identifying the vendor.    I'm not sure if this will be sufficient, but I cannot really think of another way to express it.